nginx.conf

```diff
@@ -34,6 +34,12 @@ server {
         rewrite ^/(.*[^/])$ /$1/ permanent;
     }
 
+    # Don't cache status check
+    location ~ ^/_status {
+      try_files $uri $uri.html $uri/ =404;
```

Comment:
@nottrobin  I couldn't find the way not to duplicate this line, so maybe there is a better way to write this rule.